plugins/modules/inventory_intent.py

```diff
@@ -190,7 +190,11 @@
         type: bool
         default: False
       credential_update:
-        description: Make this as true needed for the updation of device credentials and other device details.
+        description: Make this as true needed for the updation of device credentials and other device details. Also if this parameter
```

Comment:
'True' or true?

I have revised the statements. Could you please review the revised version below? If you find it to be clearer and more concise, could you address it? Otherwise, please disregard it.

`Set this to 'True' to update device credentials and other device details. When this parameter is 'True', ensure that the devices are present in Cisco Catalyst Center; only then can update operations be performed on the respective devices. If the parameter is 'True' and any device is not present, the module will attempt to add it.  If required parameters are missing during this addition, the module will fail and stop execution, preventing update operations for devices that are already present.`

Reply:
updated.